psi4/src/psi4/libfock/jk.h

```diff
@@ -1191,7 +1175,118 @@ class PSI_API MemDFJK : public JK {
     std::shared_ptr<DFHelper> dfh() { return dfh_; }
 };
 
+/**
+ * Class SplitJKBase
+ *
+ * A wrapper class that allows building of different J and K algorithms
+ * (such as Direct-DF J builds, and LinK builds)
+ */
+class SplitJKBase {
```

Comment:
Here is the header information for the base class in which split J/K algorithms would inherit from